### DISCUSSION TITLE
Circular Star Tracks

### DISCUSSION THREAD

**Discussion by PLANETARAS** [Stellarium/stellarium#3544] [General]
Hello everyone. I'm trying to make a script to visualize the circular tracks of the stars during the 24 hours. Through the marker and editing the Analemma script, I was able to visualize the Arcturus track. Would it be possible to select multiple stars at once to view tracks together? Does anyone know how to select the seven stars of the Big Dipper

**Comment by alex-w** [comment]
Same way, just use a few markers ;)

**Reply by PLANETARAS** [reply]
Thank you, great success, I will create a beautiful script with your advice
![Tracce UMA](https://github.com/Stellarium/stellarium/assets/97767104/a62a1a8f-97e6-470c-890d-a14516e9ebde)

**Reply by gzotti** [reply]
Interesting. You can probably remove stuff around labelWeek.
We (@10110111 , @alex-w) should maybe come to a solution on #1888...

**Reply by PLANETARAS** [reply]
I work by trial and error and it's very interesting. Yes, in fact, already done. Here's the script :

core.selectObjectByName("Dubhe", false);
StelMovementMgr.setFlagTracking(false);
core.selectObjectByName("Merak", false);
StelMovementMgr.setFlagTracking(false);
core.selectObjectByName("Megrez", false);
StelMovementMgr.setFlagTracking(false);
core.selectObjectByName("Phecda", false);
StelMovementMgr.setFlagTracking(false);
core.selectObjectByName("Alioth", false);
StelMovementMgr.setFlagTracking(false);
core.selectObjectByName("Mizar", false);
StelMovementMgr.setFlagTracking(false);
core.selectObjectByName("Alkaid", false);
StelMovementMgr.setFlagTracking(false);
core.wait(2);

for (i=0; i<1440; i++)
{
    var minute = i + 1;
    core.setDate("+1 minute");
    var info = core.getObjectInfo("Dubhe");
    var az = info.azimuth;
    var alt = info.altitude;
    MarkerMgr.markerHorizon(az, alt, true, "gear", "#ffffff", 1);

    core.wait(.01);    

    var info = core.getObjectInfo("Merak");
    var az = info.azimuth;
    var alt = info.altitude;
    MarkerMgr.markerHorizon(az, alt, true, "gear", "#ffffff", 1);

    core.wait(.01);    

 var info = core.getObjectInfo("Phecda");
    var az = info.azimuth;
    var alt = info.altitude;
    MarkerMgr.markerHorizon(az, alt, true, "gear", "#ffffff", 1);

    core.wait(.01);    

 var info = core.getObjectInfo("Megrez");
    var az = info.azimuth;
    var alt = info.altitude;
    MarkerMgr.markerHorizon(az, alt, true, "gear", "#ffffff", 1);

    core.wait(.01);    

 var info = core.getObjectInfo("Alioth");
    var az = info.azimuth;
    var alt = info.altitude;
    MarkerMgr.markerHorizon(az, alt, true, "gear", "#ffffff", 1);

    core.wait(.01);    

 var info = core.getObjectInfo("Mizar");
    var az = info.azimuth;
    var alt = info.altitude;
    MarkerMgr.markerHorizon(az, alt, true, "gear", "#ffffff", 1);

    core.wait(.01);    

 var info = core.getObjectInfo("Alkaid");
    var az = info.azimuth;
    var alt = info.altitude;
    MarkerMgr.markerHorizon(az, alt, true, "gear", "#ffffff", 1);

    core.wait(.01);    

}

**Reply by alex-w** [reply]
Just try it:
```
var stars = new Array("Dubhe", "Merak", "Megrez", "Phecda", "Alioth", "Mizar", "Alkaid");
StelMovementMgr.setFlagTracking(false);
core.wait(0.1);
for (i=0; i<240; i++)
{
	core.setDate("+1 minute");
	core.wait(.01); 
	for (j=0; j<stars.length; j++) 
	{
		var info = core.getObjectInfo(stars[j]);
		MarkerMgr.markerHorizon(info.azimuth, info.altitude, true, "disk", "#ffffff", 1);
	}
}
```

**Reply by PLANETARAS** [reply]
FANTASTIC!!
